playground/backend/containers/go/Dockerfile

```diff
@@ -69,16 +69,16 @@ ENV HTTP_PROXY="http://127.0.0.1:8081"
 ENV HTTPS_PROXY="http://127.0.0.1:8081"
 
 COPY entrypoint.sh /
-COPY proxy.sh /
-# Run proxy
-RUN sh /proxy.sh
 
 # Create a user group `appgroup` and a user `appuser`
 RUN groupadd --gid 20000 appgroup \
   && useradd --uid 20000 --gid appgroup --shell /bin/bash --create-home appuser
-# Chown all the files to the app user.
-RUN chown -R appuser:appgroup /opt/playground/backend/ && chown -R appuser:appgroup /opt/playground/prepared_folder/ \
-  && chmod +x /entrypoint.sh
+
+RUN mkdir -p /opt/playground/backend/executable_files/
+
+# Chown all required files to the `appuser`.
+RUN chown -R appuser:appgroup /opt/playground/backend/executable_files/ \
+  && chmod -R 777 /usr/local/share/ca-certificates/extra/ && chmod -R 777 /etc/ssl/certs && chmod +x /entrypoint.sh
```

Comment:
for this folder /usr/local/share/ca-certificates/extra/  /etc/ssl/certs needs right execute  (7)    ?

Reply:
if set `776` for the `/usr/local/share/ca-certificates/extra/` and `/etc/ssl/certs` then I receive:
```
cp: cannot stat '/usr/local/share/ca-certificates/extra/mitmproxy-ca.crt': Permission denied
/usr/sbin/update-ca-certificates: 114: cd: can't cd to /etc/ssl/certs
```

if set `776` for the `/etc/ssl/certs` then I receive:
```
/usr/sbin/update-ca-certificates: 114: cd: can't cd to /etc/ssl/certs
```

if set `776` for the `/usr/local/share/ca-certificates/extra/` then I receive:
```
cp: cannot stat '/usr/local/share/ca-certificates/extra/mitmproxy-ca.crt': Permission denied
```
So I guess I need to set `777` for both of them.